bdb/cursor.c

```diff
@@ -4553,10 +4561,19 @@ static int bdb_cursor_move_and_skip_int(bdb_cursor_impl_t *cur,
 
         /* If we've found something, see if we should skip it. */
         if (IX_FND == rc) {
+            if (is_tran_sosql(cur->ifn->impl))
```

Comment:
we need this for deduping single pass deletes

Reply:
If I'm reading bdb_tran_deltbl_isdeleted() correctly, it only does any work for non sosql txns.